sys/net/gnrc/network_layer/sixlowpan/ctx/gnrc_sixlowpan_ctx.c

```diff
@@ -131,7 +136,11 @@ gnrc_sixlowpan_ctx_t *gnrc_sixlowpan_ctx_update(uint8_t id, const ipv6_addr_t *p
 
 static uint32_t _current_minute(void)
 {
+#if IS_USED(MODULE_ZTIMER_MSEC)
+    return ztimer_now(ZTIMER_MSEC) / (MS_PER_SEC * 60);
```

Comment:
```suggestion
    return ztimer_now(ZTIMER_MSEC) / (MS_PER_SEC * SEC_PER_MIN);
```

(if you want you can also adapt the line below)